### ISSUE TITLE
Add aliases for Expect@1.x in the transformer

### ISSUE THREAD

**Issue by Justkant** [skovhus/jest-codemods#74]
Hi,

It would be nice to take into account the aliases for every functions that are map in the transformer for Expect.

Ex: https://github.com/mjackson/expect#toinclude

I don't know if there would be some collision problems because of the same function name present but with different behavior.

**Comment by skovhus** [comment]
Hi @Justkant 

Thanks for reporting this. I thought we already did this... Do you have a specific case that is not transformed?

**Comment by Justkant** [comment]
Yes maybe it's really special but we were using the old `toContain` alias of `toInclude` ([link](https://github.com/mjackson/expect#toinclude)) with an object as parameter and I think that we should use the new `toContainEqual` ([link](https://facebook.github.io/jest/docs/en/expect.html#tocontainitem)) for this case.

**Comment by skovhus** [comment]
@Justkant thanks. Didn't now these old alias. Let us add these to jest-codemods. : )

**Comment by Justkant** [comment]
Hi @skovhus 
Yes but `toContain` in some cases cannot be used as the old one.
Expect was using the [`isEqual`](https://github.com/ljharb/is-equal) module for `toContain` and so it was doing a deep equality check on objects and arrays.
This is not the case anymore, the new `toContain` is doing a strict equality check.
I think that we should use [`toContainEqual`](https://facebook.github.io/jest/docs/en/expect.html#tocontainitem) in those situations.

**Comment by skovhus** [comment]
@Justkant is this not relevant anymore? Else let us keep it open. : )

**Comment by Justkant** [comment]
Oh yes, my bad.

We used `jest-codemods` to update our tests. Because some of them were using `toContain` (the old alias) we had to manually edit them.
And so now we don't need really the support for those aliases but you're right that it could be useful to other people.

Thanks for your work on this.

**Comment by skovhus** [comment]
Sure. I'm happy it was useful to you! : ) 

Let me know if you find other issues or have ideas for improving the codemod.
